src/cli.ts

```diff
@@ -4,22 +4,35 @@ import {convert} from './convert.js';
 import {input} from '@inquirer/prompts';
 
 const args = process.argv.slice(2);
-const enableDebug = args.includes('--debug');
+const options = args.filter(arg => arg.startsWith('--'));
+const unknownOptions = options.filter(arg => arg !== '--debug');
 
-let tsConfigFilePath = '';
+if (unknownOptions.length) {
+  console.error(`Unknown options: ${unknownOptions.join(', ')}`);
+  process.exit(1);
+}
+
+const enableDebug = options.includes('--debug');
 
-try {
-  tsConfigFilePath = await input({default: 'tsconfig.json', message: 'Enter the file path to your TS config'});
-} catch {
-  // Capturing "Ctrl + C" in "Inquirer" prompts
-  console.log('Goodbye!');
+const configFiles = args.filter(arg => !arg.startsWith('--'));
+
+if (configFiles.length === 0) {
+  try {
+    configFiles.push(await input({default: 'tsconfig.json', message: 'Enter the file path to your TS config'}));
+  } catch {
+    // Capturing "Ctrl + C" in "Inquirer" prompts
+    console.log('Goodbye!');
+  }
 }
 
-if (tsConfigFilePath) {
+configFiles.forEach(tsConfigFilePath => {
+  if (enableDebug) {
+    console.log(`Processing: ${tsConfigFilePath}`);
```

Comment:
Do you expect many configs to be passed in? If not, then I think it's fine to always log the TS Config path. This way we won't need to introduce a `--debug` option.

Reply:
I have a mono repo with around 150. I was going to do a big pass. ... And logging would be okay for that.
Afterwards I was considering some enforcement on the repo and I didn't know if that would be done as a big list or smaller steps. I think either way just logging it is going to be fine.
Will adjust.